File: web3rpc/rpc-specs/code-samples/python/kaia/others/getStakingInfo.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

blockTag = "latest"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.get_staking_info(blockTag)

print(kaia_response)

File: web3rpc/rpc-specs/code-samples/javascript/kaia/others/getStakingInfo.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");

  const blockNumberOrTag = "latest";

  provider.kaia
    .getStakingInfo(blockNumberOrTag, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
